apps/mobile/ios/Podfile.lock

```diff
@@ -2484,7 +2484,7 @@ SPEC CHECKSUMS:
   Mixpanel-swift: 2192b9a24cf41b870749e4e0e10fbc9822bb2b4a
   MixpanelReactNative: 76314cf184a17d493e2dd8a3e367c81d91361eea
   OneSignalXCFramework: f5b2a3c4f130e4c910ead7bb25bed7455e976fbf
-  RCT-Folly: bf5c0376ffe4dd2cf438dcf86db385df9fdce648
+  RCT-Folly: 84578c8756030547307e4572ab1947de1685c599
```

Comment:
I think if one or both of us cleared our pod cache we might be able to stop the ping pong here, though it doesn't hurt me really and needs no attention. Just silly.